ovh/resource_cloud_project_database.go

```diff
@@ -327,9 +377,10 @@ func waitForCloudProjectDatabaseDeleted(client *ovh.Client, serviceName, engine
 
 			return res, res.Status, nil
 		},
-		Timeout:    20 * time.Minute,
-		Delay:      5 * time.Second,
-		MinTimeout: 3 * time.Second,
+		Timeout:      30 * time.Minute,
```

Comment:
are you sure this is 95%  enough ?

Reply:
Most action will be completed under 20 minutes, 30 minutes is needed for flavor upgrade or node addition.